openedx/core/djangoapps/edx_global_analytics/utils.py

```diff
@@ -192,3 +193,96 @@ def platform_coordinates(city_name):
     Module `pytz` also has list of cities with `pytz.all_timezones`.
     """
     return get_coordinates_by_platform_city_name(city_name) or get_coordinates_by_ip()
+
+
+def request_exception_handler_with_logger(function):
+    """
+    Request Exception decorator. Logs error message if it exists.
+    """
+    def request_exception_wrapper(*args, **kwargs):
+        """
+        Decorator wrapper.
+        """
+        try:
+            return function(*args, **kwargs)
+        except requests.RequestException as error:
+            logger.exception(error.message)
+            return
+
+    return request_exception_wrapper
+
+
+def get_access_token():
+    """
+    Return single access token for authorization and dispatch installation statistics to OLGA acceptor.
+    """
+    try:
+        access_token = AccessTokensStorage.objects.first().access_token
```

Comment:
Please add docstring, that it is working only for single acceptor per edx instance, not for multiple acceptors. (If I understood correctly.)